### PR TITLE
[ODH-2262]: Thinkanalytics httpaccess logparsing updated with IDs

### PR DESCRIPTION
**Pull request by nilan3**
Include traxis-profile-id, request-id and crid fields in thinkanalytics http access log parsing step.

### CODE REVIEW

**Comment by TARGETrus** [comment]
expected results should be changed in 'src\test\it\resources\log_parsing\expected_result.txt'
to fit new 'src\applications\log_parsing\think_analytics\event_with_url_creator.py' logic

File: src/test/unit/applications/log_parsing/think-analytics/test_think_analytics_log_parsing.py [hunk 1]
```diff
@@ -16,20 +16,54 @@ def test_httpaccess(self):
             {
                 "topic": "vagrant_in_eosdtv_lab5aobo_tst_heapp_thinkanalytics_httpaccess_log_v1",
                 "source": "localhost_access_log",
-                "message": "[19/Jun/2017:02:06:17 +0200] 172.30.189.59 http-0.0.0.0-8080-64 GET /RE/REController.do?contentSourceId=1&clientType=399&method=lgiAdaptiveSearch&subscriberId=278e3270-4d12-11e7-85f5-e5a72ae6734d_nl HTTP/1.1 200 14"
+                "message": "[04/Apr/2018:16:47:05 +0200] 10.95.96.119 http-0.0.0.0-8080-5 GET /RE/REController.do?clientType=300&actionId=3&subscriberId=5ec2ed6f-a5c4-4afe-8970-39b9c57fc39c_be#MasterProfile&contentSourceId=1&actionTime=1522853226&method=learnAction&contentItemId=crid://telenet.be/e36c8aef-2934-45bd-ae22-244c8e796d6d&contentItemInstanceId=imi:001000000020B7F0 HTTP/1.1 200 5"
```
Comment (vunder-kind) [inline src/test/unit/applications/log_parsing/think-analytics/test_think_analytics_log_parsing.py]: Change contentItemId to check your parsing for splitting

**Comment by vunder-kind** [comment]
Update basic analytics for think_analytics after changing names for some fields.

**Comment by nilan3** [comment]
@vunder-kind No changes are required on the think_analytics basic analytic job as they do not make use of fields `subscriberId`, `contentItemId`, `intRequestId`

**Comment by vunder-kind** [comment]
@nilan3 We have subscriberId in schema creation for httpaccess_driver. Just remove it, otherwise we might have some errors in future. Also it will be great to rename all subscriberId in input file in resources for httpaccess_test, because if it be used for some extensions of analytics jobs, we also will have an error.

**Pull request merged**